src/VisualStudio/Core/Def/HACK_ThemeColorFixer.cs

```diff
@@ -117,6 +117,9 @@ public void RefreshThemeColors()
             UpdateForegroundColor(ClassificationTypeNames.TypeParameterName, sourceFormatMap, targetFormatMap);
             UpdateForegroundColor(ClassificationTypeNames.ModuleName, sourceFormatMap, targetFormatMap);
 
+            UpdateForegroundColor(ClassificationTypeNames.TestCode, sourceFormatMap, targetFormatMap);
+            UpdateBackgroundColor(ClassificationTypeNames.TestCode, sourceFormatMap, targetFormatMap);
```

Comment:
no clue if i need to do this.

Reply:
By adding it, it does make the method name a little suspect. From looking over the code it seems to be used to update colors in the tooltip format map (quick info, signature help), so maybe if this will be rendered in a tooltip. Likely won't hurt.